cssFiles/input.css

```diff
@@ -0,0 +1,107 @@
+
+.outer-boundary{
+    /* border-style: solid;
+    border-width: thin; */
+    margin: .5rem;
+    padding: .5rem;
+    border-radius: .5rem;
+    
+    
+  }
+  .label-position{
+    width: 50%;
+    
+  }
+  
+  .user-input {
+      padding: 0.5rem;
+      width: 50%;
+      border: 1px solid grey;
+      border-radius: 0.2rem;
+      margin: .5rem;
+     
+  }
+  
+  .form-content{
+    
+    display: flex;
+    flex-direction: column;
+     align-items: center;
+    
+  }
+  
+  #login-btn{
+    width: 30%;
+    background-color: rgb(20, 176, 228);
+    color: white;
+    padding: .5rem;
+    margin: .5rem;
+    border-radius: .5rem;
+    
+  }
+  
+  #login-btn:hover{
+    background-color: darkblue;
+  }
+  
+  #create-btn{
+    width: 30%;
+    background-color: white;
```

Comment:
suggestion - set colors in variable.css file and use var here

Reply:
ok